docs/achitecture_and_design/api_design.md

```diff
@@ -13,22 +13,20 @@ The following are the RESTful endpoints needed to support basic chat and follow-
         {
             "user_profile": {
                 "id": 1,
-                "url": "http://pennyuniversity.org/api/users/1",
+                "url": "http://pennyuniversity.org/api/users/profiles/1",
                 "real_name": "Joe",
                 "user": {
                     "id": 1,
```

Comment:
Is it necessary to include a user here? If a participant is represented in a chat by their user profile, then I don't think we need to expose their user profile. Usernames that are used for authentication should be protected.

Reply:
Exposing usernames is probably a minor concern though

Reply:
@CoreyAR Maybe we just expose the ID, or a url to the user endpoint? I was thinking it would be important to know the user for some frontend permissions like exposing edit buttons.

Reply:
Ahh, I see. I need to double check but reviewing further down the document, it seems that followups are attributed to `user_profiles` so it is worth thinking if permissions should be based on `user_profiles` or `users`

Reply:
Basing it on `user_profiles` could be tricky since there eventually could be multiple user profiles for a single user.